src/library_glut.js

```diff
@@ -289,7 +289,7 @@ var LibraryGLUT = {
         document.removeEventListener('mozfullscreenchange', GLUT.onFullscreenEventChange, true);
         document.removeEventListener('webkitfullscreenchange', GLUT.onFullscreenEventChange, true);
       }
-      Browser.setCanvasSize(width, height);
+      Browser.setCanvasSize(width, height, true);
```

Comment:
the `, true` additions in this patch are not entirely clear to me. what is the reason there?

Reply:
`GLUT.reshapeFunc` is registered as a canvas resize callback at https://github.com/kripken/emscripten/blob/1.37.39/src/library_glut.js#L364. Without the `, true` addition, the reshape callback is called in `Browser.setCanvasSize` at L292 and then again immediately after this line. It's not bug here, but I think the duplication just makes no sense.

See also:
- https://github.com/kripken/emscripten/commit/998106173842d2e6510368ed0ef325fe78370475
- https://github.com/kripken/emscripten/commit/3ef19c9e22c2417e95542ae7f31432944b26c0d5

Reply:
I see, thanks. Makes sense. How about adding a comment that this is already a resize callback, and therefore we don't need to send updates? Then the `, true` is less surprising to readers.

Reply:
Done.